src/main/java/com/gotocompany/firehose/utils/KafkaConnectorType.java

```diff
@@ -0,0 +1,14 @@
+package com.gotocompany.firehose.utils;
+
+import lombok.Getter;
+
+@Getter
+public enum KafkaConnectorType {
```

Comment:
Yes, initially I was thinking of applying it to the Kafka consumer as well, so trying to name this enum as generic as possible. What if we call it `KafkaConnectorTypesMetadata` so that in future we can also add consumer as well if needed?